app/facades/trust_registry.py

```diff
@@ -50,3 +145,19 @@ async def get_did_for_actor(actor_id: str) -> List[str]:
     did = actor_res.json()["did"]
     didcomm_invitation = actor_res.json()["didcomm_invitation"]
     return [did, didcomm_invitation]
+
+
+async def register_schema(schema_id: str) -> None:
+    schema_res = requests.post(
+        TRUST_REGISTRY_URL + "/registry/schemas", json={"schema_id": schema_id}
+    )
+
+    if schema_res.status_code != 200:
+        raise Exception(f"Error registering schema {schema_id}: {schema_res.text}")
+
+
+async def register_actor(actor: Actor) -> None:
+    actor_res = requests.post(TRUST_REGISTRY_URL + "/registry/actors", json=actor)
```

Comment:
Does
```python
json=actor
```
possibly  have to be 
```python
json=actor.dict()
```
or 
```python
json=json.dumps(actor.dict())
```

Reply:
`Actor` is a typed dict, not a pydantic model. So it is already a dict. I have tested it and it works fine.